docs/addition-tips.md

```diff
@@ -0,0 +1,42 @@
+---
+sidebar_position: 6
+---
+
+
+# Additional Tips and Tricks
+
+**Boundaries:**
+
+The Meta Quest 3 may prompt the user through a series of brief steps to create a boundary or enable via passthrough. It is recommended to create a boundary \> stationary boundary for use at an office desk.
+
+**Trouble seeing the Vitessce Desktop UI through passthrough?**
+
+Enhance font sizes by moving your head closer to the screen, or increasing your browser size on Mac Command+ and on Windows STRG \+. You can reset the size after the session by using  on Mac Command+0 and on Windows STRG+0
+
+**Hands or Controller?**
+
+Combined use of both controllers and hand gestures may sometimes cause the Meta Quest 3 to not use the one you want for control input. This issue is usually related to wanting to use your hands for gestures while the controllers are still powered on. In this case, try removing the batteries from the controllers to help the Meta Quest 3 ‘forget’ about that input and allow easy use of hand gestures.
+
+**Working Environment:**
+
+The Meta Quest 3 works passthrough works best in natural light environments. If the light is dim or artificial it can reduce the quality of the camera passthrough and the hand tracking ability of the goggles.
+
+[image1]: <data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABQAAAAUCAIAAAAC64paAAACf0lEQVR4Xn3UV2/TUBQH8PuxEEIgscVSERTEEB0glmgzmtE4y47jpM5qVtNQZhGgAGIJqJAQr2AnnrHjJHwW1HCc6xjjVtyHyC//3N85Psfo+8avr83hp8bgbd14VR8+q/QfrvaaBb2W14uslmW1VEaJM0qYln2UvEhJt5PyfFy6RoiXIgL61hxsNQcf1gZv6oOX1f5m2XhQ0teLeiWv5Vk9s6KRmW4sLYdTKoTvJaVbCWkuJl4hhAuRDoJrvzSG7+v9Vs14UTGelHsbq3q90Cvn9FxOT690k4waTavBlOIhpbtJ6UZcnIkJl6PC9HLbDGNzqzZwmVlWo7NqIqOCOUCZYWyeiUpgPhfuIJf5UdlwmLtO8yKlOM1nw230ed00v671/5qLGpjzbJehDYpWomkZzF5SWkh3neapEI8mZuN52TTfL/bAPNpxPKRyJwFmAcLYfCbUQR8bQ2zGBYPZnRuNwAy/TjOETwfbyGVuFDRnLM78ILIGfgbzbEyGMDafDHbQuzXzJcG1j1d1bC7lrMuhzwSxZf/RfIJzmk8E2sjqc9UyQ58LOW3cZ2V7+7edhDMbU65GRdt8LMAjMEP4acXAZvySmGzXGbPCcbNg23xkiUdg3izrUDCYYbDADLPhzo0PNk9HxKmx+bCfQ63JMjjN7txoBH0G80XCLBjMx4Ptgz4e7WqGwYJ5jpCqJ2Uuw03o88QMgwXmo37eDO80M6xm9plWJsug2IN1flnAZij4gI9DtrlaMJcBzNYCj+d5gRTxMrjMh5a4/V4egRkWuFnq/bPAtBqiFNcCY/OpgGXe5+GQy/yfBXaZ93o5tLuZlv0pBZuvx6yPjsu8x/vzD7UhDL3CP6c9AAAAAElFTkSuQmCC>
```

Comment:
Could we import these images the way they're embedded elsewhere, e.g. `![A screenshot of the Vitessce Link Page](./assets/images/loginVit.png)` on `tutorial.md`? The inline image data files are pretty bulky to read, don't allow for the same optimizations to image loading to be performed by the browser that separate files do, and don't currently have useful alt text.

Reply:
thanks for pointing out, these were orphan links from the initial markup download from google drive. Removed now.